data/governments.txt

```diff
@@ -41,6 +41,84 @@ government "Builder"
 		destroy 0
 		atrocity 0
 
+government "Bunrodea"
```

Comment:
Just double-checking -  you have to *destroy* a ship to trigger the mission, yes? The intro missions say you have *attacked* a faction, which isn't necessarily the same thing.

I'd suggest either using `provoke`, but that depends on what the Queen would consider "being attacked" - if the Korath attack(ed) the Bunrodea a lot, perhaps just firing on a vessel is normal to them and not considered an actually attack.

If not, I suggest clarifying in the intro mission that you *destroyed* a (insert bunrodean faction's vessel here), not attacked one. Also keeps in mind that the guard attacks you - if your response is "they attacked first", then it makes even more sense for the queen to say you *destroyed* them, if that's what it takes to trigger the mission.

Reply:
If I'm confusing this with a default `provoke` value I'm not considering, it's worth noting me firing on the Royal Guard hasn't triggered the mission it seems.

Reply:
The Queen does say you've already taken care of the ones that attacked you, if you choose one of the options
But yes this could be clarified in general

Reply:
Yes, but the leading line from the Queen is "You *attacked*'", regardless if it's clarified later, that should be more clear as the first line from an alien.

Reply:
Is it false that you attacked a Bunrodean ship? You're only summoned if you destroyed one, but saying you attacked one isn't less true, it's just less specific.

Reply:
I suppose it's coming from a later thought - if people aren't sure about how to start the Bunrodea mission chain, and are told that attacking Bunrodea causes it, they might get confused if they don't actually destroy them. Some people will murder everything just because, but some others may hesitate and only think firing is enough. Just some food for thought.

Reply:
I think it's fine as is. Note that in the future, this will not be the only means of first contact.